x/staking/spec/01_state.md

```diff
@@ -15,6 +15,21 @@ Store entries prefixed with "Last" must remain unchanged until EndBlock.
 
 - LastTotalPower: `0x12 -> ProtocolBuffer(sdk.Int)`
 
+## ValidatorUpdates
```

Comment:
is this optional? normal staking chains shouldn't need to store this data

Reply:
Without it the call to `GetValidatorUpdates`, which is exposed by the staking keeper, will fail.